src/components/auth/UserMenu.jsx

```
@@ -0,0 +1,203 @@
+              <ListItemIcon>
+                <AccountCircle fontSize="small" />
+              </ListItemIcon>
+              <ListItemText>Create Account</ListItemText>
+            </MenuItem>
+          </>
+        )}
+      </Menu>
+
+      <AuthDialog open={authDialogOpen} onClose={closeAuthDialog} initialView={authDialogView} />
+
+      <style jsx global>{`
```

ESLint

[error] Unknown property 'jsx' found

(react/no-unknown-property)